src/main/resources/index.mustache

```diff
@@ -4,6 +4,7 @@
     <a href="/facebook/index.html">Protected url by Facebook: facebook/index.html</a> (use a real account)<br />
     <a href="/facebook/notprotected.html">Not protected page: facebook/notprotected.html</a> (no authentication required)<br />
     <a href="/form/index.html">Protected url by form authentication: form/index.html</a> (use jerome/jerome, the credentials are checked on jumpcloud.com)<br />
+    <a href="/basicauth/index.html">Protected url by indirect basic auth: basicauth/index.html</a> (use jle/password)<br />
```

Comment:
Is it in this commit? I can't find the code you are referring to, sorry, I think I'm missing something ^^